### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by AleMdta**
In this Milestone:                         
1.Add Linters Configurations.                        
2.Add README.md File.            
3.Add HTML and CSS Semantics.                          
 Thanks for your time for review my PR. :)

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+"This file should be ignored by git".
```
Comment (Johnadibe) [inline test.md:2]: > - [ ] Please kindly make sure that the test.md file is actually ignored by git and is not present in your GitHub repository as stated in the project requirement👇.☺
>> - _Make sure that the last file is actually ignored by git and is not present in your GitHub repository._

File: README.md [hunk 2]
```diff
@@ -245,39 +137,12 @@ Feel free to check the [issues page](../../issues/).
...
+> 
 
-  - [Answer_2]
+If you like this project wait to see more. ;)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
```
Comment (Johnadibe) [inline README.md:148]: > - [ ] The link to your MIT license is broken. Please kindly fix that😊. This makes you to be the owner of your project.
> 
> mit

**Review by Johnadibe — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️
### Hi @AleMdta 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Highlights
✅ Linter checks passed✔️
✅ Meaningful commit messages 👍🏽
✅ Added .html and .css files 👍🏽


## Required Changes ♻️
> - [ ] I noticed that the pull request you submitted didn't include a very detailed description. It's important to write a descriptive pull request that explains what changes you've made, why you made them, and how they benefit the project. This helps other team members review your code more effectively and understand the purpose of the changes you've made. Please kindly include a detailed description in your pull requests👇.👍 

alemdta-description

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @AleMdta    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive Readme✔️





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**